### DISCUSSION TITLE
Welcome to Detours Discussions!

### DISCUSSION THREAD

**Discussion by bgianfo** [microsoft/Detours#206] [Announcements]
## Welcome!!! 🤟🥳
  We’re using Discussions as a place to connect with other members of the Detours community. We hope that you:
* Ask questions!
* Ask for help on problems you're facing!
* Share thoughts and ideas on how to make Detours itself or the Detours community better!
* Engage with other community members!

We request that everybody follow the [Microsoft Code of Conduct](https://opensource.microsoft.com/codeofconduct/). We are creating a welcoming community, where everyone feels comfortable sharing their questions and experience, no matter who they are or how long they've been using Detours. We treat everyone with respect here.

  To get started, comment below with an introduction of yourself and tell us how you use Detours!

**Comment by igupta-microsoft** [comment]
Hello all, my name is Ishita and I have recently started working with Detours for our NodeJs service. I want to hook Detour to the winsock2 APIs connect() and closeSocket(). The Detour is implemented in a .cc file which is compiled into a binary that is loaded into a typeScript class using "require". Now the typeScript class has a method that does a "fetch" call and I am expecting the Detour to attach to the connect() call that will happen in the backgroun of fetch() and replace it with my function. But I do see that the attachment is not happening. Can someone please give me some pointers as to what I might be doing wrong?
